app/Rules/Uppercase.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+
+class Uppercase implements ValidationRule
+{
+    /**
+     * Run the validation rule.
+     *
+     * @param  \Closure(string): \Illuminate\Translation\PotentiallyTranslatedString  $fail
+     */
+    public function validate(string $attribute, mixed $value, Closure $fail): void
+    {
+        if ($value[0] !== strtoupper($value[0])) {
+            $fail('The title does not start with an uppercased letter');
```

Comment:
replace `title` to `:attribute` in string to make most effective to use in another attribute.

keep going 👍